pkg/pricing/models.go

```diff
@@ -5,157 +5,157 @@ import (
 )
 
 type Quote struct {
-	CreatedAt   *time.Time `json:"createdAt" bson:"createdAt"`
-	ExternalIDs []string   `json:"externalIDs,omitempty" bson:"externalIDs,omitempty"`
+	CreatedAt   *time.Time `json:"createdAt"`
+	ExternalIDs []string   `json:"externalIDs,omitempty"`
 	// includes
 	// salesforce:opportunity:a3HVT000001HYBB2A4
 	// salesforce:plan:a3JVT000001Kh532AC
-	ID             string          `json:"quoteID" bson:"quoteID"`
```

Comment:
i'd remove the comments above just to keep it clean. if a user really needs more details they can always look at the pricing-api